src/content/docs/en/guides/on-demand-rendering.mdx

```diff
@@ -53,11 +53,11 @@ export async function GET() {
 ```
 ### `'server'` mode
 
-For a **highly dynamic app**, after adding an adapter, [set your build output configuration to `output: 'server'`](/en/reference/configuration-reference/#output) to **server-render all your pages by default**. This is the equivalent of opting out of prerendering on evey page.
+For a **highly dynamic app**, after adding an adapter, [set your build output configuration to `output: 'server'`](/en/reference/configuration-reference/#output) to **server-render all your pages by default**. This is the equivalent of opting out of prerendering on every page.
 
 Then, if needed, you can choose to prerender any individual pages that do not require a server to execute, such as a privacy policy or about page.
 
-```astro title="src/pages/about-my-app.astro"
+```astro title="src/pages/about-my-app.astro" {2}
```

Comment:
Same concern here about the highlighting. The next example uses "highlighting" but we don't use it here. Maybe we should use `ins` here and in the next example too? (I just updated from the next example)

Reply:
Sure, let's make them all `ins`!